topics/howtosim.rst

```diff
@@ -0,0 +1,255 @@
+you set ``ignore = false``, it may take a moment to generate the cylindrical bases cache file. The cylindrical bases will be
+saved out to whatever name is specified in ``cachefile``, and the halo basis will generally be saved out to
+``.slgrid_sph_cache``. Note that the ``diskgen`` YAML file here is distinct from the EXP YAML config file, EXP YAML config files 
+look like :ref:`this annotated example <yamlconfig>`.
+
+Step Two: generate cache files
+******************************
+If you didn't generate cache files in step one, you need to generate these files now. An example of how to do this is in the
+`pyEXP-Examples repo <https://github.com/EXP-code/pyEXP-examples/blob/main/How-To/Utilities/create%20Cylinder%20basis%20(parallel).py>`_.
+If you are including a stellar disk in your simulation, EXP requires that you provide a cylindrical cache file with a name that matches
+the ``cachename`` for the **star disk** in the EXP YAML config file. You are not required to provide a spherical cache file, as it is much 
+faster to compute the spherical cache file than the cylindrical cache file. You must specify a name for the spherical cache file in the 
+EXP YAML config file, and if that file does not exist, EXP will generate the spherical cache file on-the-fly and save it as the specified name.
+
+Step Three: three-part relaxation process (**optional**)
+*********************************************************
+We recommend a performing the following three step process to produce quiet initial conditions. This process minimizes ringing and other symptoms of
+slight dynamical disequilibrium in the initial conditions by allowing the disk and halo to adjust and virialize in the presence of one another before
+starting the full simulation. The steps are as follows:
+
+A. After creating your disk and halo body files, relax the halo in the presence of the disk. To do this, set the following parameters in the ``force`` parameter map for the halo portion of the EXP YAML config to true:
+
+.. code-block:: python
+   
+   self_consistent:       true
+   EVEN_L:                true
+   M0_ONLY:               true
+
+and set ``self_consistent`` in the ``force`` parameter map of the disk portion of the EXP YAML config to false
+
+.. code-block:: python
+   
+   self_consistent:       false
+
+(see e.g. :ref:`an annotated example <yamlconfig>`). The rest of your config YAML file should be as normal. Lets call this YAML file "first_step_config.yml". We suggest running for at least T ~ 4 Gyr, though you should check the output coefficients to know if you should run for longer. Note that in the YAML config, you can reduce the cadence of when phase space dumps (i.e. snapshots) are saved out to avoid unnecessarily cluttering up your storage. The ``acyl`` and ``hcyl`` parameters should match ``ascale`` and ``hscale`` in your ``gendisk`` YAML config file. Run EXP with this YAML config file:
```

Comment:
In practice, all one really needs is `outchkpt` with `nint` set to the number of steps for the entire simulation.   In other words, I'd chuck out `outpsp` our `outpsq` altogether.